app/src/main/java/com/duckduckgo/app/survey/api/SurveyRepository.kt

```diff
@@ -33,6 +34,12 @@ interface SurveyRepository {
     fun remainingDaysForShowingSurvey(survey: Survey): Long
     fun shouldShowSurvey(survey: Survey): Boolean
     fun getScheduledSurvey(): Survey
+    fun getScheduledLiveSurvey(): LiveData<Survey>
```

Comment:
This is a late comment, but as you were touching this it'd have been great to get rid of this `LiveData`